va_explorer/templatetags/va_explorer_tags.py

```diff
@@ -11,6 +12,27 @@
 
 register = template.Library()
 
+@register.filter
+def replace(value):
+    value = value.strip()
+    if("dk" == value or "DK" == value):
+        return "Don't know"
+    elif("nan" == value):
+        return "n/a"
+    elif("veryl" == value):
+        return "Very Low"
+    elif("ref" == value):
+        return "refuse to answer"
+    else:
```

Comment:
Can you make the capitalization consistent with the other replacements, i.e., "Refuse to Answer"